src/translations/pl/translation.json

```diff
@@ -260,20 +260,20 @@
     "Chance": "Szansa",
     "Count": "Ilość",
     "map": "mapa",
-    "Spawn chance": "Szansa pojawienia",
+    "Spawn chance": "Szansa pojawienia się",
     "Chance that the boss spawns on a given map": "Szansa, że boss pojawi się na danej mapie",
-    "Health": "Życie",
+    "Health": "PŻ",
```

Comment:
Why shorten Health to HP? We have enough space in properties to use full-length words.

Reply:
Mostly for clarity I guess; Health is a name of a skill and HP would make it clear it's the health points pool.

Reply:
The game too show Health at title label:

![Heath](https://user-images.githubusercontent.com/10927011/216836872-238073cb-a745-43fb-8743-1c455161b416.jpg)

Reply:
Good point, for some reason I used the Życie instead of Zdrowie, fixed that. :)